### ISSUE TITLE
withWebComponentsKnobs does not show "Knobs" Tab on page load

### ISSUE THREAD

**Issue by peschee** [open-wc/open-wc#1837]
Currently, there seems to be an issue with the `withWebComponentsKnobs` decorator. On page load, the "Knobs" tab is not shown. Only after changing between stories in the navigation, the knobs tab is shown.

This issue can be replicated / seen in the official example. When navigating to

https://open-wc.org/demoing-storybook/?path=/docs/decorators-withwebcomponentknobs--example-output

the knobs tab is not shown. After switching to another story and then back, the knobs tab shows. Here's an example recording of the issue:

![Screen-Recording-2020-09-21-at-12 42 04](https://user-images.githubusercontent.com/63866/93758119-1f1aa800-fc08-11ea-9b1d-abb79cfb4fc5.gif)

**Comment by peschee** [comment]
The same (or similar) issue is also with the `<sb-props>` web component that shows the API derived from `custom-elements.json`. This is also only available after navigating around and not upon page load:

![Screen-Recording-2020-09-21-at-12 47 36](https://user-images.githubusercontent.com/63866/93758574-ff37b400-fc08-11ea-870d-48fcd85c408b.gif)

**Comment by peschee** [comment]
@LarsDenBakker @daKmoR @thepassle Any pointers on where to look so I can come up with a PR?

**Comment by LarsDenBakker** [comment]
Sorry there hasn't been any response here. It's like a race condition. However with storybook v6 there is actually a built-in replacement for this in storybook itself. We're working that as part of https://modern-web.dev/docs/dev-server/plugins/storybook/

Again really sorry this issue was left gather dust... what do you think about moving to storybook v6?

**Comment by peschee** [comment]
@LarsDenBakker I will give it a shot and let you know.

**Comment by peschee** [comment]
@LarsDenBakker Is mdjs supported with the new dev server + storybook v6?

**Comment by daKmoR** [comment]
we recommend using `@web/dev-server` and its storybook v6 plugin with the new knobs system.

**Comment by peschee** [comment]
@daKmoR Do you have any concrete examples of that setup? I know the docs page https://modern-web.dev/docs/dev-server/plugins/storybook/ but the open-wc repo + the old setup had a more complete example (e.g. the card component and its documentation).

**Comment by daKmoR** [comment]
@paales you can take a look at the official storybook docs
https://storybookjs.netlify.app/web-components-kitchen-sink/?path=/story/addons-knobs--simple

**Comment by peschee** [comment]
Oh oh, no we dragged a „@paales“ into this ;) Thanks @daKmoR, will look into that.

**Comment by daKmoR** [comment]
oups sorry 😅

**Comment by paales** [comment]
Haha, 'what the hell is this?!" haha, no worries
